bigflow/cli.py

```diff
@@ -485,20 +516,23 @@ def _resolve_vault_endpoint(args):
         return None
 
 
-def _resolve_property(args, property_name):
-    cli_atr = getattr(args, property_name)
-    if cli_atr:
-        return cli_atr
-    else:
-        config = import_deployment_config(_resolve_deployment_config_path(args), property_name)
-        return config.resolve_property(property_name, args.config)
+def _resolve_property(args, property_name, ignore_value_error=False):
+    try:
+        cli_atr = getattr(args, property_name)
+        if cli_atr:
+            return cli_atr
+        else:
+            config = import_deployment_config(_resolve_deployment_config_path(args), property_name)
+            return config.resolve_property(property_name, args.config)
+    except ValueError:
```

Comment:
Why do you want to allow ignoring missing properties?

Reply:
Now missing value 'vault_secret' is ignored (honestly not sure why)
https://github.com/allegro/bigflow/blob/f2d9008b1e1eb653e7e8035a0a0b0cc623be0c9f/bigflow/cli.py#L498
this PR just doesn't change that

Reply:
👍